src/web-ui/post_build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# This script is called during the build stage in CodePipeline.
+# It deploys web-ui app on Amazon CloudFront.
+# If $XDN_DEPLOY_TOKEN is present it deploys the app on Moovweb XDN.
+
+if [ $XDN_DEPLOY_TOKEN = "NONE" ]; then
+  echo Uploading to ${WEB_BUCKET_NAME}
+  aws s3 cp --recursive ./dist s3://${WEB_BUCKET_NAME}/ 
+  aws s3 cp --cache-control="max-age=0, no-cache, no-store, must-revalidate" ./dist/index.html s3://${WEB_BUCKET_NAME}/
+  aws cloudfront create-invalidation --distribution-id ${CLOUDFRONT_DIST_ID} --paths /index.html
+else
+  echo Uploading to XDN
+  npx xdn deploy --skip-build --token=$XDN_DEPLOY_TOKEN
+fi
```

Comment:
This logic means that webapp is no longer being deployed to CloudFront. I assume the idea is that the entire webapp will run in XDN and that CloudFront does not serve as the origin for XDN? If so, that will break the instructions throughout the documentation that tell the user to use the WebURL output parameter from CloudFormation to access the deployed web app. 

Does it make sense to still deploy the webapp to CloudFront even when XDN is enabled so the user can access the site either through XDN or CloudFront? For example, the user could do a side by side performance comparison between the two. I'm not sure if that makes sense or not, though, since the CloudFront instance would still be pointing at XDN endpoints for the microservices, right? I think I need to better sense of how XDN works and what would make for the best user experience for the Retail Demo Store.